boilermaker/retries.py

```diff
@@ -4,7 +4,7 @@
 
 from pydantic import BaseModel, field_validator
 
-MAX_REASONABLE_RETRIES = 30
+MAX_REASONABLE_RETRIES = 100
```

Comment:
This is higher than I would have expected.

Reply:
yeah, I bumped it up so I could make a crazy example in my tests. Then I realized that someone might want to retry something a whole lot and this is a hard-cap on constructing one of these (would have to subclass in order to get around the validator), so it seemed fine to set a higher max?

Reply:
It's tough, one the one hand I'm thinking "why are we enforcing a restriction here on these values" on the other hand it seems like a safe default as I could see myself easily fatfingering this to an unreasonable number. The maximum and the default both being 100 seems high. The comment below states the default retry is 5 which seems like a more sensible default?

Reply:
oh, I see. Yeah, I didn't meant to make the default `max_tries` so high. That was a mistake that didn't surface because all of my tests use the `default()` constructor. I will set a different default! I agree that having a max in case I fat-finger this is probably a good idea.